files/build_templates/docker_image_ctl.j2

```diff
@@ -106,7 +106,13 @@ function preStartAction()
             echo -n > /tmp/dump.rdb
             docker cp /tmp/dump.rdb database$DEV:/var/lib/redis/
         fi
+        # Add redis UDS to the redis group and give read/write access to the group
+        REDIS_SOCK="/var/run/redis${DEV}/redis.sock"
+    else
+        REDIS_SOCK="/var/run/redis-chassis/redis_chassis.sock"
     fi
+    chgrp -f -R redis "$(dirname "$REDIS_SOCK")" && \
+    chmod -f -R 0775 "$(dirname "$REDIS_SOCK")"
```

Comment:
>0775 [](http://example.com/codeflow?start=16&length=4)

Should it be 760?

Reply:
`/var/run/redis/` needs 775 for sufficient permission to get Redis start if `/var/run/redis/` is owned by group `redis`. `/var/run/redis/sonic-db/` need 755 if owned by `root` group or 775 if `/var/run/redis/sonic-db/` is owned by `redis` group to pass kvm testcase `iface_namingmode/test_iface_namingmode.py`

Reply:
Done